common/service/config/config.go

```diff
@@ -180,6 +182,21 @@ type (
 		RootCAData []string `yaml:"rootCaData"`
 	}
 
+	// WorkerTLS contains TLS configuration for system workers within the Temporal Cluster to connect to Temporal frontend.
+	WorkerTLS struct {
+		// The path to the file containing the PEM-encoded public key of the certificate to use.
+		CertFile string `yaml:"certFile"`
```

Comment:
[Nit] to avoid confusion, can we call this "clientCertFile, clientKeyFile, etc"

Reply:
Renaming would create inconsistency with other such fields in the Internode section. I think we better do that as part of a future restructuring of the TLS config.
For now I added a clarification to the code comments that these are for a client cert.